brianlong/www.validators.app
# 184205913 - add authorities to vote account view

#### What's this PR do?
PR adds the authorized withdrawers and voters to the vote account details page.

#### How should this be manually tested?
- evaluate the design of new components and its content
- verify it also on various resolutions (web, mobile)
- check if the new content appears when there aren't any records
- both authorized withdrawers and voters should be displayed in descending order as well as they should be unique.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184205913)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
